### ISSUE TITLE
cluster/kube-up.sh don't work, while KUBERNETES_PROVIDER is centos

### ISSUE THREAD

**Issue by xiaoping378** [kubernetes/kubernetes#32192]
**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):

**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.):

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

**Kubernetes version** (use `kubectl version`):

v1.3.6

**Environment**:
- **Cloud provider or hardware configuration**:
X86 lenovo PC

- **OS** (e.g. from /etc/os-release):
[root@node0 ~]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

- **Kernel** (e.g. `uname -a`):
Linux node0 3.10.0-327.el7.x86_64 #1 SMP Thu Nov 19 22:10:57 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

**What happened**:
run the below cmd, error.
```
export KUBERNETES_PROVIDER=centos 
cluster/centos/build.sh all    ----      error
```

**What you expected to happen**:

Can't download docker binary.

I have fix it, and default install the latest kubernetes, docker, flannel, etcd.

**Comment by asalkeld** [comment]
If #30847 fixes the problem can we close this issue?

**Comment by xiaoping378** [comment]
fixed by #30847
